src/common/qprocessordetection.h

```diff
@@ -0,0 +1,390 @@
+// Copyright (C) 2016 The Qt Company Ltd.
```

Comment:
I liked the previous layout more where this stuff is in a `tools` directory. And where the defines come through CMake rather than direct inclusion. I don't want to include all these Q_xxx macros which might even conflict with some Quake stuff.

Reply:
Yeah, I tried to reuse the header in code itself but I went back from this. I haven't moved them back to `tools/` yet but none of the two added headers are actually included in compiled code.

Reply:
Moved back to `tools/`.